_scripts/requirements.txt

```diff
@@ -1,2 +1,3 @@
 compare-locales ~= 9.0.2
```

Comment:
Let's not; I'm hoping to drop the c-l dependency in subsequent PRs. Also, `~=` includes 9.0.4 so that's what we're already using in CI: https://github.com/mozilla-l10n/firefox-l10n-source/actions/runs/9830946288/job/27137710120#step:5:13